### DISCUSSION TITLE
Disable type hints in method override signatures

### DISCUSSION THREAD

**Discussion by zotil** [microsoft/pylance-release#2698] [Enhancement]
Hello,

I don't know if it's the right place to ask, if not please let me know where to go.

Actually I am working with Django and I am overriding a class method, then it show autocomplete which is very good, the point is that I am not working with type hints and the method gets autocompleted with the hints and classes that are not imported

When I type:
> def save_mo...
trying to reach save_model

It autocompletes like this:
```python
def save_model(self, request: Any, obj: _ModelT, form: Any, change: Any) -> None:
        return super().save_model(request, obj, form, change)
```

expected autocompletion:
```python
def save_model(self, request, obj, form, change):
        return super().save_model(request, obj, form, change)
```

Is there a way to config this?
Thank you!

**Comment by jakebailey** [comment]
We currently don't have a configuration for this, no. We're including the types because we include stubs for Django, which includes the types for these parameters (so we carry them down).

**Comment by zotil** [comment]
Thank you for the answer. Would be great if we could configure it in the future.

**Comment by nambread** [comment]
Adding my own two cents that I'd also love this added. For my particular use case I'm using type comments in the PEP 484 style due to working on a python 2.7 project, I even have a Python 2.7 interpreter set in the VS Code workspace, and still the autocompletes bring in the type hints in the python 3.7 style, which isn't valid syntax.

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
